protoc_plugin/lib/src/file_generator.dart

```diff
@@ -685,12 +695,12 @@ const _fileIgnores = {
   'annotate_overrides',
   'camel_case_types',
   'constant_identifier_names',
+  'deprecated_member_use',
   'directives_ordering',
   'library_prefixes',
   'non_constant_identifier_names',
   'prefer_final_fields',
   'return_of_invalid_type',
```

Comment:
Not related to this PR, but I think we can remove this ignore as well. I tried removing it locally and it doesn't generate any warnings in test protos.

Reply:
I'll remove in this PR (I verified what you were saying locally as well)